org.kde.akregator.json

```diff
@@ -1,10 +1,10 @@
 {
     "id": "org.kde.akregator",
     "runtime": "org.kde.Platform",
-    "runtime-version": "6.7",
+    "runtime-version": "6.9",
```

Comment:
This one does not exists yet.

Reply:
Will rebuild when its available :+1:

Reply:
I don't see the point in having this 3 months open in advance